nextcord/types/channel.py

```diff
@@ -155,3 +155,6 @@ class StageInstance(TypedDict):
     topic: str
     privacy_level: PrivacyLevel
     discoverable_disabled: bool
+
+class GuildDirectory:
+    type: Literal[14]
```

Comment:
```suggestion
    type: Literal[14]

```

Reply:
would be nice to have a newline at the end, pep8

Reply:
Done